laplace/curvature/asdl.py

```diff
@@ -63,7 +63,7 @@ def gradients(self, x, y):
         Gs : torch.Tensor
             gradients `(batch, parameters)`
         """
-        f = batch_gradient(self.model, self.lossfunc, x, y).detach()
+        f = batch_gradient(self.model, self.lossfunc, x, y)
```

Comment:
Does the `detach()` also have to be removed from for the Jacobians [here](https://github.com/AlexImmer/Laplace/blob/main/laplace/curvature/asdl.py#L42)?

Reply:
Fixed in [6e50ab4](https://github.com/AlexImmer/Laplace/pull/120/commits/6e50ab4e774b2834b11aa5e984bdd1c069255b49)